oc_config_validate/demo/certs/create_certs.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
```

Comment:
just point to: https://github.com/google/gnxi/tree/master/certs

?

Reply:
Thanks for the feedback. Ack, I now have the run_demo.sh script get and run this script from raw github.